sphinx/tutorial/rose/furthertopics/rose-bunch.rst

```diff
@@ -16,11 +16,11 @@ Rather than creating multiple apps or
 
```

Comment:
There is an unwanted blank line here causing a new paragraph to begin for the text in 17:20.